packages/puppeteer-extra-plugin-recaptcha/package.json

```diff
@@ -13,29 +13,24 @@
   "author": "berstend",
   "license": "MIT",
   "scripts": {
-    "clean": "rm -rf dist/*",
+    "clean": "rimraf dist/*",
     "tscheck": "tsc --pretty --noEmit",
-    "build": "yarn clean && tsc --project tsconfig.json --module commonjs && rollup -c rollup.config.ts && yarn ambient-dts",
-    "docs": "exit 0",
-    "docs2": "rm -rf docs/* && typedoc --module commonjs --readme none --target ES6 --theme markdown --excludeNotExported --excludeExternals --excludePrivate --out docs --mode file src/index.ts",
-    "test": "TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' ava -v src/**.test.ts",
-    "test-ci": "echo pwd && yarn build && TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\"}' ava --fail-fast -v dist/*.test.js",
-    "ambient-dts": "yarn ambient-dts-copy && yarn ambient-dts-fix-path",
-    "ambient-dts-copy": "copyfiles -u 1 \"src/**/*.d.ts\" dist",
-    "ambient-dts-fix-path": "replace-in-files --string='/// <reference path=\"../src/' --replacement='/// <reference path=\"../dist/' 'dist/**/*.d.ts'"
```

Comment:
Did we accidentally lose `ambient-dts-fix-path` here? It's unfortunately needed to make TypeScript pick up our ambient declarations

Reply:
I removed that, because there were no files that ever matched that regex..

Reply:
> I removed that, because there were no files that ever matched that regex..

You mean `dist/**/*.d.ts`? This will only exist/be populated after the build step (ts to js). :-)

Reply:
The `.d.ts` files that will be copied to `dist/` and need to be patched can be seen here: https://github.com/berstend/puppeteer-extra/tree/master/packages/puppeteer-extra-plugin-recaptcha/src

Reply:
You are right. I did check to see if there were any `reference path` references in `.d.ts` files, but didn't see any, because I was relying on a human sight :)

Then I used `grep` and found that indeed there is a reference.

Fixed!